public/index.js

```diff
@@ -20,7 +20,7 @@
       "boring",
       "kind",
       "sad"
-    ].forEach(emotion => {
+    ].forEach(function(emotion){
```

Comment:
Ops, this is my bad, defo something I should have picked up! Sorry about that (spending too long in node land)